recipes/mkl_random/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.1" %}
+{% set buildnumber = 0 %}
+
+
+package:
+    name: mkl_random
+    version: {{ version }}
+
+source:
+    url: https://github.com/IntelPython/mkl_random/archive/v{{version}}.tar.gz
+    md5: f37e16fd040cb28c6cfc726aa68cd8fd
+
+build:
+    number: {{buildnumber}}
+    skip: True  # [win and py27]
+
+requirements:
+    build:
+      - python
+      - setuptools
+      - mkl-devel
+      - cython
+      - numpy 1.11.*
+    run:
+      - python
+      - mkl
+      - numpy >=1.11
+
+test:
+    commands:
+      - nosetests -v mkl_random
+    requires:
+      - nose
+    imports:
+      - mkl_random
+      - mkl_random.mklrand
+
+about:
+    home: http://github.com/IntelPython/mkl_random
+    license: BSD
+    license_file: LICENSE.txt
+    summary: NumPy-based implementation of random number generation sampling using Intel (R) Math Kernel Library, mirroring numpy.random, but exposing all choices of sampling algorithms available in MKL.
```

Comment:
This is a bit long. Could you please shorten it and/or wrap it? Ideally the `summary` is only one line. Would take a look at using `description` if you have more info that you want to share.